examples/lock-app/linux/include/LockEndpoint.h

```diff
@@ -36,19 +36,22 @@ struct LockUserInfo
 struct LockCredentialInfo;
 struct WeekDaysScheduleInfo;
 struct YearDayScheduleInfo;
+struct HolidayScheduleInfo;
```

Comment:
Why are these forward-declared where as LockUserInfo is declared in place?

Reply:
Uh, no particular reason except I didn't want to clutter the top of the file with complete declarations.